web-wallet/src/lib/containers/MigrateContract/MigrateContract.svelte

```diff
@@ -171,29 +164,24 @@
       screenWidth = entry.contentRect.width;
     });
 
-    (async () => {
-      if (isConnected) {
-        await walletDisconnect();
```

Comment:
I hav no explanation of why this was here. Any ideas? cc: @HDauven

Reply:
I remember @deuch13 had some kind of corner case here, maybe he can tell us what it was.
The async IIFE instead is just to avoid the top-level await.

Reply:
The walletConnect interface does not always disconnect from the web-wallet UI when the user closes or refreshes the tab with the web-wallet. This will result in instances where the third-party wallet will still be connected to the users web-wallet but he will not be logged in it. This is the reason for that check. @nortonandreev @ascartabelli